test/proxy/ERC1967/ERC1967Proxy.test.js

```diff
@@ -9,5 +9,5 @@ contract('ERC1967Proxy', function (accounts) {
     return ERC1967Proxy.new(implementation, initData, opts);
```

Comment:
The calls to `createProxy` in `Proxy.behavior.js` no longer include the proxyAdminAddress. Shouldn't the arguments of this function reflect that ?

Reply:
Yes. This is fixed now. For the record, the reason I removed the admin/initialOwner from the arguments of this function was that the base Proxy behavior shouldn't need to be aware of the initial owner.